includes/tpl-install-instructions.md

```diff
@@ -1,2 +1,2 @@
 > [!NOTE]
-> The TPL Dataflow Library (the <xref:System.Threading.Tasks.Dataflow> namespace) is not distributed with the .NET Framework. To install the <xref:System.Threading.Tasks.Dataflow> namespace, open your project in Visual Studio, choose **Manage NuGet Packages** from the **Project** menu, and search online for the `System.Threading.Tasks.Dataflow` package.
+> The TPL Dataflow Library (the <xref:System.Threading.Tasks.Dataflow> namespace) is not distributed with .NET. To install the <xref:System.Threading.Tasks.Dataflow> namespace in Visual Studio, choose **Manage NuGet Packages** from the **Project** menu, and search online for the `System.Threading.Tasks.Dataflow` package. Alternatively, to install it using [the .Net Core CLI](~/docs/core/tools/index.md), run `dotnet add package System.Threading.Tasks.Dataflow`.
```

Comment:
The **Manage NuGet Packages** option is only available if a project is open. Removing the phrase creates the impression that the TPL Dataflow Library can be installed without a project being open.

Reply:
@rpetrusha Done.